### PR TITLE
uyuni: Use default HTTP client and set relative paths

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@o11y.eu>

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -989,12 +989,13 @@ var expectedConf = &Config{
...
-					Password:        "hole",
-					Entitlement:     "monitoring_entitled",
-					Separator:       ",",
-					RefreshInterval: model.Duration(60 * time.Second),
+					Server:           "https://localhost:1234",
+					Username:         "gopher",
+					Password:         "hole",
+					Entitlement:      "monitoring_entitled",
+					Separator:        ",",
+					RefreshInterval:  model.Duration(60 * time.Second),
```
Comment (davetweetlive) [inline config/config_test.go:997]: A suggestion: Take 60  as constant somewhere in the package

File: discovery/uyuni/uyuni.go [hunk 2]
```diff
@@ -52,9 +52,10 @@ const (
 
 // DefaultSDConfig is the default Uyuni SD configuration.
 var DefaultSDConfig = SDConfig{
-	Entitlement:     "monitoring_entitled",
-	Separator:       ",",
-	RefreshInterval: model.Duration(1 * time.Minute),
+	Entitlement:      "monitoring_entitled",
+	Separator:        ",",
+	RefreshInterval:  model.Duration(1 * time.Minute),
```
Comment (davetweetlive) [inline discovery/uyuni/uyuni.go:57]: The same  constant can be  used here

Comment (roidelapluie) [reply discovery/uyuni/uyuni.go:57]: I don't think a constant is really needed here.

Comment (Nexucis) [reply discovery/uyuni/uyuni.go:57]: I don't think too.
For consistency with the test, maybe choose `1 * time.Minute` or `60 * time.Second` but not both even if it doesn't really matter

**Review by Nexucis — APPROVED** [review]
LGTM

**Pull request merged**